src/test/javascript/general.controller.test.js

```diff
@@ -1,4 +1,4 @@
-describe("GeneralController", function () {
```

Comment:
I do not know Jasmine well enough to comment extensively on this so it would be beneficial for someone else to take a look at this section in addition to me. (I would love to learn Jasmine @William-Liang808 @devgav please message me if you would be willing to go over this code with me).